meta-iot2000-example/scripts/lib/wic/canned-wks/example-image.iot2000.wks

```diff
@@ -6,7 +6,7 @@ part --source efibootguard-efi --size 32 --extra-space 0 --overhead-factor 1 --o
 part / --source rootfs --size 1536 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --fstype=ext4 --label platform0 --align 1024
 part --source efibootguard-boot --size 32 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label boot0 --align 1024 --part-type=0700 --sourceparams "watchdog=60,revision=2,initrd=acpi-upgrades-iot2000.cpio"
 part swap --ondisk mmcblk0 --size 512 --fstype=swap --label swap --align 1024
-part --size 1024 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label persistent --align 1024 --fstype=ext4
+part /data --size 1024 --extra-space 0 --overhead-factor 1 --ondisk mmcblk0 --label persistent --align 1024 --fstype=ext4
 
 bootloader --ptable gpt --append="console=ttyS1,115200n8 earlyprintk initrd=acpi-upgrades-iot2000.cpio"
 
```

Comment:
This structuring of patches is a bit too fine-grained. Fold with the patches that creates the data partition.

Reply:
Okay folded them.